util/task.py

```diff
@@ -17,6 +18,8 @@
 from util.pkv_calculation import define_base_values, calc_pkv
 
 
+log_file_path = path.join(path.dirname(path.abspath(__file__)), 'logger_configuration.conf')
+logging.config.fileConfig(log_file_path, disable_existing_loggers=False)
 logger = logging.getLogger('util/task')
```

Comment:
Just out of curiosity. What is the difference between naming the logger explicitely and `logging.getLogger(__name__)` like in the other files? Maybe unify it?